airmozilla/manage/tests/views/test_regions.py

```diff
@@ -0,0 +1,67 @@
+from nose.tools import eq_, ok_
+
+from funfactory.urlresolvers import reverse
+
+from airmozilla.main.models import (
+    Region,
+)
+from .base import ManageTestCase
+
+
+class TestRegions(ManageTestCase):
+    def test_regions(self):
+        """Region management pages return successfully."""
+        response = self.client.get(reverse('manage:regions'))
+        eq_(response.status_code, 200)
+
+        region = Region.objects.create(name='New Region')
+        response = self.client.get(reverse('manage:regions'))
+        eq_(response.status_code, 200)
+        eq_(region.name, 'New Region')
```

Comment:
Add another test to list 1 inactive region. E.g. 
```python
def test_regions_inactive(self):
    Region.objects.create(name='New Region', is_active=False)
    response = self.client.get(reverse('manage:regions'))
    eq_(response.status_code, 200)
    ok_("Not active!" in response.content)
```
(note: I can't remember what the little label says next to the region name if it's not active).